mafContrib/GRBTransientMetric.py

```diff
@@ -4,40 +4,99 @@
 import lsst.sims.maf.metrics as metrics
 import numpy as np
 
-__all__ = ['GRBTransientMetric']
-
-class GRBTransientMetric(metrics.TransientMetric):
-    """Detections for on-axis GRB afterglows decaying as
+class GRBTransientMetric(metrics.BaseMetric):
```

Comment:
Can you add the 
`__all__ = ['GRBTransientMetric'] `

back into the file please? 
This line says that this is the only thing to be exported from this file. It's probably not strictly necessary here, but is a generally good idea to add to the files in the sims_maf_contrib package, to avoid exporting things into the mafContrib namespace that were not intended to be there.